src/components/SvgIcon/SvgIcon.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <svg>
-    <use :xlink:href="`#${name}`" />
+    <use :xlink:href="`#${props.name}`" />
```

Comment:
Why so? I guess we better remove the `const props =` part, though I don't think it's even related.

Reply:
The props definition is required for receiving props afaik. If we don't use props ESLint will complain, it's an explicit way of reading the props.